.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: false
+
+language: node_js
+node_js:
+    - '5.0'
```

Comment:
The minor version doesn't need to be specified, so just `- 5` will do, it will download the latest v5 rather than always the first one.

But really I don't think this is needed. We can probably go with whatever Travis defaults to